internal/api/v1/application/create.go

```diff
@@ -95,7 +97,14 @@ func Create(c *gin.Context) apierror.APIErrors {
 	var routes []string
 	if createRequest.Configuration.Routes != nil {
 		// Note: Routes can be empty here!
-		routes = createRequest.Configuration.Routes
+		for _, d := range createRequest.Configuration.Routes {
+			// Strip scheme prefixes, if present
+			routeURL, err := url.Parse(d)
+			if err == nil && routeURL.Scheme != "" {
+				d = strings.TrimPrefix(d, routeURL.Scheme+"://")
+			}
```

Comment:
Since we are parsing the URL we should also check for the err (it should not fail but still...):

```suggestion
			routeURL, err := url.Parse(d)
			if err != nil {
				return apierror.NewBadRequestError(err.Error()).WithDetails("failed to parse route")
			}
			if routeURL.Scheme != "" {
				d = strings.TrimPrefix(d, routeURL.Scheme+"://")
			}
```

Reply:
ok, commit e5c46e789fa2ccdeb3fd9554f74658ca5bce21e4